OutOfSchool/OutOfSchool.WebApi/Extensions/ElasticsearchSynchronizationExtension.cs

```diff
@@ -32,44 +33,14 @@ public static class ElasticsearchSynchronizationExtension
 
             if (elasticSynchronizationSchedulerConfig.UseQuartz)
```

Comment:
if `elasticSynchronizationSchedulerConfig` is controlled by the end user - it is possible it might be null, so it should be initialized with some default value in case of `null`, otherwise NRE will be thrown